sdk-installer/src/run.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+import subprocess
+from typing import Optional
+
+from velocitas_lib import (
+    get_programming_language,
+    get_project_cache_dir,
+    get_workspace_dir,
+)
+
+SUPPORTED_LANGUAGES = ["cpp"]
+
+
+def get_required_sdk_version_cpp() -> Optional[str]:
+    """Return the required version of the for C++.
+
+    Returns:
+        Optional[str]: The required version or None in case SDK is not a dependency.
+    """
+    sdk_version: Optional[str] = None
+    with open(
+        os.path.join(get_workspace_dir(), "conanfile.txt"), encoding="utf-8"
+    ) as conanfile:
+        for line in conanfile:
+            if line.startswith("vehicle-app-sdk/"):
+                sdk_version = line.split("/", maxsplit=1)[1].split("@")[0].strip()
+
+    return sdk_version
```

Comment:
Duplicate of function in conan_helper.py

Reply:
We can consider introducing a generic version and moving it to velocitas_lib, but I'd prefer not to do this now.